### PR TITLE
feat(pages): duplicate echo to novu-framework

### PR DESCRIPTION
**Pull request by justnems**
Prepping for tearing this apart a bit in order to build out the new homepage.

### CODE REVIEW

File: src/components/pages/novu-framework/code/code-with-tooltip/code-with-tooltip.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import StepTooltip from './step-tooltip';
+
+const TOOLTIP_CONTENT = {
+  stepTooltip: <StepTooltip />,
+  echoTooltip: (
+    <span className="flex flex-col py-3.5">
+      <span className="whitespace-normal px-3.5">
+        (alias) <span className="text-yellow">new Echo</span>(&#123; apiKey, ... opts &#125;?:
```
Comment (unicodeveloper) [inline src/components/pages/novu-framework/code/code-with-tooltip/code-with-tooltip.jsx:10]: I see Echo here? Should this be Client instead?

..to match the new syntax/way of doing things.

File: src/components/pages/novu-framework/code/code.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+import createCustomElement from './create-custom-element';
+
+const CODE = `import { Echo } from '@novu/echo';
+
+const echo = new @@Echo@echoTooltip@@();
+
+const commentWorkflow = echo.workflow('comment-on-post', async (@@event@eventTooltip@@) => {
+  const inAppResponse = await event.step.inApp('notify-user', async () => ({
+    body: renderReactComponent(event.@@payload@payloadTooltip@@.postId)
+  }));
```
Comment (unicodeveloper) [inline src/components/pages/novu-framework/code/code.jsx:16]: All of this still uses the old way. Should we rewrite to use the new way?

File: src/components/pages/novu-framework/code/code-with-tooltip/code-with-tooltip.jsx (hunk 1 above)
Comment (justnems) [reply src/components/pages/novu-framework/code/code-with-tooltip/code-with-tooltip.jsx:10]: I'll change the code snippet once we have a new one from @scopsy. Just spoke w/ him about that this morning.

File: src/components/pages/novu-framework/code/code.jsx (hunk 2 above)
Comment (justnems) [reply src/components/pages/novu-framework/code/code.jsx:16]: Same comment as above. :)

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**